src/utils/init/utils.js

```diff
@@ -0,0 +1,136 @@
+const path = require('path')
+
+const chalk = require('chalk')
+const cleanDeep = require('clean-deep')
+const dotProp = require('dot-prop')
+const inquirer = require('inquirer')
+const isEmpty = require('lodash/isEmpty')
+
+const { fileExistsAsync, writeFileAsync } = require('../../lib/fs')
+
+const normalizeDir = ({ siteRoot, dir, defaultValue }) => {
+  if (dir === undefined) {
+    return defaultValue
+  }
+
+  const relativeDir = path.relative(siteRoot, dir)
+  return relativeDir || defaultValue
+}
+
+const getDefaultSettings = ({ siteRoot, config }) => {
+  const defaultBuildCmd = dotProp.get(config, 'build.command')
+  const defaultBuildDir = dotProp.get(config, 'build.publish')
+  const defaultFunctionsDir = dotProp.get(config, 'build.functions')
+
+  return {
+    defaultBuildCmd,
+    defaultBuildDir: normalizeDir({ siteRoot, dir: defaultBuildDir, defaultValue: '.' }),
+    defaultFunctionsDir: normalizeDir({ siteRoot, dir: defaultFunctionsDir, defaultValue: 'functions' }),
+  }
+}
+
+const getBuildSettings = async ({ siteRoot, config }) => {
+  const { defaultBuildCmd, defaultBuildDir, defaultFunctionsDir } = getDefaultSettings({ siteRoot, config })
+  const { buildCmd, buildDir, functionsDir } = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'buildCmd',
+      message: 'Your build command (hugo build/yarn run build/etc):',
+      filter: (val) => (val === '' ? '# no build command' : val),
+      default: defaultBuildCmd,
+    },
+    {
+      type: 'input',
+      name: 'buildDir',
+      message: 'Directory to deploy (blank for current dir):',
+      default: defaultBuildDir,
+    },
+    {
+      type: 'input',
```

Comment:
Previously a log was shown with the value of the functions directory (if exists).
Now we always prompt the user to set it - we could also only prompt to update it if it exists